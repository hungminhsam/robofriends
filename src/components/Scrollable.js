import React from "react";

const Scrollable = (props) => {
  return (
    <div
      style={{
        overflowY: "scroll",
        borderTop: "1px solid grey",
        height: "800px",
      }}
    >
      {props.children}
    </div>
  );
};

export default Scrollable;
