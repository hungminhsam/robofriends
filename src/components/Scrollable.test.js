import { shallow } from "enzyme";
import React from "react";
import Scrollable from "./Scrollable";

describe("<Scrollable />", () => {
  it("Snapshot Test for Scrollable", () => {
    expect(shallow(<Scrollable />).debug()).toMatchSnapshot();
  });

  it("render children when passed in", () => {
    const wrapper = shallow(
      <Scrollable>
        <div className="unique" />
      </Scrollable>
    );

    expect(wrapper.contains(<div className="unique" />)).toEqual(true);
  });
});
