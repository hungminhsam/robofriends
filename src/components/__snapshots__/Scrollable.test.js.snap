// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scrollable /> Snapshot Test for Scrollable 1`] = `"<div style={{...}} />"`;
